pkg/scheduler/plugins/elasticquota/quota_overuse_revoke.go

```diff
@@ -0,0 +1,282 @@
+/*
+Copyright 2022 The Koordinator Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package elasticquota
+
+import (
+	"sync"
+
+	"github.com/emirpasic/gods/sets/treeset"
+	v1 "k8s.io/api/core/v1"
+	quotav1 "k8s.io/apiserver/pkg/quota/v1"
+	"k8s.io/component-helpers/scheduling/corev1"
+	"k8s.io/klog/v2"
+	"k8s.io/kubernetes/pkg/api/v1/resource"
+
+	"github.com/koordinator-sh/koordinator/apis/extension"
+	"github.com/koordinator-sh/koordinator/pkg/scheduler/plugins/elasticquota/core"
+)
+
+type QuotaOverUsedGroupMonitor struct {
+	groupQuotaManger                  *core.GroupQuotaManager
+	quotaName                         string
+	overUsedContinueCount             int64
+	continueOverUsedCountTriggerEvict int64
+	podCache                          map[string]*v1.Pod
```

Comment:
here's podCache should be killed, use podCache in quotaInfo

Reply:
fixed